appliance_catalog/views.py

```diff
@@ -25,6 +25,8 @@
 from keystoneclient.v3 import client
 from glanceclient import Client
 from smtplib import SMTPException
+from six.moves import http_client, urllib
```

Comment:
Why this and not `requests`? Is there a standard one we use? @xandroc

Reply:
"six" is used somewhere else and already installed, but "requests" is not.